studiopress/pattern-manager
# Get the default values from `get_pattern_defaults()`

* Maybe this PR is really nitpicky, let me know 😄 
* It makes the meta defaults come from `get_pattern_defaults()`
* Doesn't change the defaults
* But as we're getting the defaults in more places, this makes sure there's a single source of truth

### How to test

1. Go to the Patterns UI
2. Click 'Create New Pattern'
3. Go to the JS console
4. Run `wp.data.select('core/editor').getEditedPostAttribute( 'meta' )`
5. Expected: 
```js
{
  blockTypes: [],
  categories: [],
  description: '',
  inserter: true,
  keywords: [],
  name: '',
  postTypes: [],
  viewportWidth: 1280
}
```
